app/src/contexts/proposal.tsx

```diff
@@ -178,7 +178,9 @@ export function ProposalProvider({ children = undefined as any }) {
         JET_GOVERNANCE
       );
       proposalsByGovernance = proposalsByGovernance.filter(
-        prop => !PROPOSAL_BLACKLIST.some(blacklisted => blacklisted.equals(prop.pubkey))
+        prop =>
+          !PROPOSAL_BLACKLIST.some(blacklisted => blacklisted.equals(prop.pubkey)) &&
+          prop.account.governingTokenMint == JET_TOKEN_MINT
```

Comment:
Use .equals() because js == can only check reference equality and there's no way to override it. Also use strict equality ===
```suggestion
          prop.account.governingTokenMint.equals(JET_TOKEN_MINT)
```